voteswap/static/javascript.js

```diff
@@ -0,0 +1,23 @@
+$(function() {
+  $(".js-propose-swap").click(function() {
+    var $button = $(this);
+    var userId = $button.data().profileId;
+    var name = $button.data().name;
+
+    var conf = confirm("Are you sure you want to propose a swap with " + name + "?");
+
+    if (conf) {
+      $button.prop("disabled", true);
+
+      $.post("/user/swap/", { to_profile: userId, csrfmiddlewaretoken: vs.csrf })
+        .done(function() {
+          $button.parents(".js-swap-option").addClass("proposed");
+        })
+        .fail(function() {
+          $button.prop("disabled", false);
+          alert("Unfortunately we were not able to confirm that swap. Please try again");
```

Comment:
I can. I'd prefer to pull that out into a separate task because its a little more complicated to manage extra divs than just popping up an alert. In theory this should never happen ;). 

Let me know if I can break it out separately and if you want me to work on that next.